versioned_docs/version-1.0/cadence-migration-guide/usdc-migration.mdx

```diff
@@ -0,0 +1,330 @@
+---
+title: USDC Migration Guide
+sidebar_position: 8
+sidebar_label: USDC Migration Guide
+---
+
+# USDC in Cadence 1.0
+
+Effective September 3, 2024, at 3 a.m. ET ahead of the Crescendo network upgrade,
```

Comment:
Can we update this paragraph to fit more this language?


> As part of the upcoming Flow Crescendo network upgrade, all ERC-20 compatible tokens on Flow can be used in Cadence smart contracts without special handling and all Cadence fungible tokens on Flow will be ERC-20 compatible. This will ensure that all Flow applications—whether they are built with Solidity, Cadence, or some mix of both—are fully interoperable with the wider web3 ecosystem.
> 
> Effective September 3, 2024, at 3 a.m. ET ahead of the Crescendo network upgrade, Circle will cease support for USDC on Cadence, as EVM-based ERC-20 USDC will become naturally compatible across the entire Flow ecosystem

We want the context to be given for the update and change. Framing can help people understand how this is an important step to the ecosystem.

Reply:
added ✅